src/pages/index.tsx

```diff
@@ -64,163 +64,37 @@ export const whatsNewQuery = groq`*[_type == 'resource' && slug.current == "what
   },
 }`
 
-export const reactFeaturesQuery = groq`*[_type == 'resource' && slug.current == "react-features"][0]{
-  title,
-  subTitle,
-  path,
-  name,
-  resources[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	},
-  related[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	}
-}
-`
-
-export const javascriptFeaturesQuery = groq`*[_type == 'resource' && slug.current == "javascript-features"][0]{
-  title,
-  subTitle,
-  path,
-  name,
-  resources[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	},
-  related[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	}
-}
-`
-
-export const cssFeaturesQuery = groq`*[_type == 'resource' && slug.current == "css-features"][0]{
-  title,
-  subTitle,
-  path,
-  name,
-  resources[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	}
-}
-`
-
-export const reduxFeaturesQuery = groq`*[_type == 'resource' && slug.current == "redux-features"][0]{
-  title,
-  subTitle,
-  path,
-  name,
-  resources[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	}
-}
-`
-
-export const typescriptFeaturesQuery = groq`*[_type == 'resource' && slug.current == "typescript-features"][0]{
-  title,
-  subTitle,
-  path,
-  name,
-  resources[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	}
-}
-`
-
-export const kcdFeaturesQuery = groq`*[_type == 'resource' && slug.current == "kent-c-dodds-features"][0]{
-  title,
-  subTitle,
-  path,
-  name,
-  resources[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	},
-  related[]->{
-    title,
-    'description': summary,
-    path,
-    'byline': meta,
-    image,
-    byline
-	}
-}
-`
-
-export const awsFeaturesQuery = groq`*[_type == 'resource' && slug.current == "aws-features"][0]{
-  title,
-  subTitle,
-  description, 
-  path,
-  name,
-  resources[]->{
+export const homePageFeatures = groq`*[_type == 'resource' && slug.current == "home-page"][0]{
+  'features': resources[]-> {
     title,
-    'description': summary,
+    slug,
+    subTitle,
     path,
```

Comment:
Currently, we are displaying "Browse more {resource.name}", with these changes it looks like we are only displaying "Browse more" -- Is that the intention? If not, adding just `name` in the query would fix that @zacjones93

Reply:
good catch!